data/json/items/armor.json

```diff
@@ -2851,7 +2851,7 @@
     "id": "fur_collar",
     "type": "ARMOR",
     "name": "fur kitty collar",
-    "description": "A decorative 'collar' made out of fur.  Complete with a little bell hanging from the front.  Don't worry, it's silent.",
+    "description": "A decorative 'collar' made out of fur, complete with a little bell hanging from the front.  Don't worry, it's silent.",
```

Comment:
Is this REALLY needed change?

Reply:
I think that there should be a discussion between the devs and translators on how correct the grammar in the game should be; because obviously if you start throwing random Punctuation and Improperly capitalized words into the mixture. Things just really do not begin to sound or look right, it's sloppy; it has a habit of taking entirely too much space and tends to find a way to become useless and descriptiveless runon.  It's choppy. It's unprofessional for a game to have improperly capitalized nouns in the only things a player sees about the item - the description.

I know you don't see half the code using lowercase kevlar and the other half uppercase Kevlar as an issue, because why would you? It gets the point across. It looks functionally the same. There's no reason to *ever* touch any of the grammar and spelling in item descriptions, just look at:

*#23556* or
*#23576* or
#25557 or
#25436 or
#25428 or
#25409 or
#25327 or
#25263 or
#25259 or
#25213 or
#25160 or
#25095 or
*#24793* or
#24611 or
*#24396* or
*#24396* or
#24316 or
#23846 or
#23780 or
#23758

For example. All of these PRs had real, functional reasons behind the change, which would obviously prevent gameplay in debilitating ways. They all REALLY needed to be changed, as you said.

I'm not trying to change every description in the game. armor.json has 10k lines of code which I went through, checked, and cross-referenced to make sure that most descriptions made sense and had a semblance of passable grammar, allowing fragments. 

The fix you commented on above was a fragment which didn't align with most of the other fragments in the descriptions. Periods. Are. Meant. To. Emphasize. Something. That's how most fragments are used in all the other descriptions. The above is just silly filler which shouldn't be emphasized, and one of the rare cases where the fragment is grammatically correct with just the punctuation change, like the author didn't know which punctuation to use in that case. The period emphasizes the wrong part of the item, which is the silent sentence.

Kevlar is a proper noun. It should be capitalized. Otherwise, why should we capitalize countries like france or poland, or the beginning of sentences, if phonetically it makes sense?

Just in the last month, there were 17 PRs added into the game which fix typos. Would the translators prefer these PRs to trickle in one at a time - only touching a few things at once - so they can get on their computers, fire up their software, and translate one or two lines a week for the rest of eternity? Or do they want to have these block-style PRs so they can sit down and crank through them? If the game is headed towards being correct grammatically and spelling-wise, which is apparent by the above PRs, then the same amount of translation work is going to be done regardless; it'll just be more *condensed* in PRs like this.

Again, up to the translators to decide if they want to do these edits every week for all of eternity or just get them out of the way.

> People has to wait for a week and play with untranslated strings

It's the experimental version. Expect breakages.